com.feaneron.Boatswain.json

```diff
@@ -126,25 +126,25 @@
             ]
         },
         {
-            "name" : "gdk-pixbuf",
-            "buildsystem" : "meson",
-            "config-opts" : [
+            "name": "gdk-pixbuf",
+            "buildsystem": "meson",
+            "config-opts": [
                 "-Dgtk_doc=false",
                 "-Dman=false",
                 "-Dothers=enabled"
             ],
-            "cleanup" : [
+            "cleanup": [
                 "/bin/*"
             ],
-            "post-install" : [
+            "post-install": [
                 "rm /app/lib/libgdk_pixbuf-2.0.so*",
                 "rm /app/lib/pkgconfig/gdk-pixbuf-2.0.pc"
             ],
-            "sources" : [
+            "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/gdk-pixbuf/2.42/gdk-pixbuf-2.42.12.tar.xz",
-                    "sha256": "b9505b3445b9a7e48ced34760c3bcb73e966df3ac94c95a148cb669ab748e3c7",
+                    "url": "https://download.gnome.org/sources/gdk-pixbuf/2.43/gdk-pixbuf-2.43.3.tar.xz",
+                    "sha256": "ed4c5d16346a5b4e95326731de9700fad70d03bbd2f9324732e6c94f42d869d7",
```

Comment:
`stable-only: true` should be added to avoid this.

See https://github.com/flathub-infra/flatpak-external-data-checker?tab=readme-ov-file#gnome-checker